cmd/sync/aws.go

```diff
@@ -131,6 +138,12 @@ func (client *AWSClient) CheckIfScalingGroupExists(name string) (bool, error) {
 
 // GetPrivateIPsForScalingGroup returns the list of IP addresses of instances of the Auto Scaling group
 func (client *AWSClient) GetPrivateIPsForScalingGroup(name string) ([]string, error) {
+	var onlyInService bool
+	for _, u := range client.GetUpstreams() {
+		if u.ScalingGroup == name && u.InService {
+			onlyInService = true
```

Comment:
Can we add a `break` here? it seems once `onlyInService` is true, is going to be true no matter what happens to the other Upstreams? Let me know if it makes sense

Reply:
ah yeah of course, good catch 😄